README.md

```diff
@@ -17,7 +17,7 @@ please reach out to your Amazon contact for access
 ## Features
 
 ### Watch Activity
-**Documentation**: TODO
+**Documentation**: https://developer.amazon.com/docs/fire-tv/watch-activity.html
```

Comment:
There's another TODO on L11, I think this is the link we'd want there: https://developer.amazon.com/docs/fire-tv/get-started-with-firetv-integration-sdk.html#step-1-include-the-sdk-in-your-app